src/worker/lambda/handlerPuller.go

```diff
@@ -33,6 +33,62 @@ type CacheEntry struct {
 	path    string // where code is extracted to a dir
 }
 
+func ccopy(src, dest string) error {
+	info, err := os.Stat(src)
+	if err != nil {
+		return err
+	}
+
+	if info.IsDir() {
+		return filepath.Walk(src, func(path string, info os.FileInfo, err error) error {
```

Comment:
The docs say "Walk is less efficient than WalkDir, introduced in Go 1.16, which avoids calling os.Lstat on every visited file or directory."  I'm not clear on the difference (hope you can look closer), but any reason WalkDir doesn't work?

Also, I'm wondering if WalkDir checks for you, or if you need to do this yourself

Reply:
Changed to WalkDir. It does say it's more efficient in the doc. Just wasn't aware this is an option when I was writing it.